lib/gcp-compute-engine.go

```diff
@@ -13,6 +13,8 @@ import (
 	"golang.org/x/oauth2/google"
 
 	"google.golang.org/api/monitoring/v3"
+	"google.golang.org/api/option"
+	ghttp "google.golang.org/api/transport/http"
```

Comment:
"google.golang.org/api/transport/http" is documented as below 👇 , and according to the document using this package is not recommended.
https://godoc.org/google.golang.org/api/transport/http
> Package transport/http supports network connections to HTTP servers. This package is not intended for use by end developers. Use the google.golang.org/api/option package to configure API clients.

Can you change not to use ghttp package?

Reply:
OMG 😖
Thank you for the notice! I'll change it.